artifactory/commands/npm/publish.go

```diff
@@ -261,8 +263,11 @@ func (npc *NpmPublishCommand) pack() error {
 		return err
 	}
 
-	npc.packedFilePath = filepath.Join(tarballDir, packageFileName)
-	log.Debug("Created npm package at", npc.packedFilePath)
+	for index, packageFileName := range packedFileNames {
+		npc.packedFilesPath = append(npc.packedFilesPath, filepath.Join(tarballDir, packageFileName))
+		log.Debug("Created npm package at", npc.packedFilesPath[index])
+	}
```

Comment:
```suggestion
	for _, packageFileName := range packedFileNames {
		npc.packedFilesPath = append(npc.packedFilesPath, filepath.Join(tarballDir, packageFileName))
	}
```
nothing is created here so it's bit odd log